src/view/items/Yielder.js

```diff
@@ -42,6 +43,14 @@ export default class Yielder extends Item {
 		}).bind();
 	}
 
+	bubble () {
+		if ( !this.dirty ) {
+			runloop.addFragment( this.fragment );
```

Comment:
For me, the tests still pass if this is commented out – which I *think* makes sense, because we're now bubbling up the correct route (through its `containerFragment`, which is responsible for calling its `update()` method). So I think we can lose this line (and the associated `import` statement)?